### ISSUE TITLE
I can't connect with the server

### ISSUE THREAD

**Issue by SimonIsMe** [mariobyn/websocket#1]
Hello, 
I run server, but when I tried to connect with them in Chrome in this way:

```
var ws = new WebSocket('ws://localhost', 8112)
```

i got an error response:

```
WebSocket connection to 'ws://localhost/' failed: Error during WebSocket handshake: Unexpected response code: 200
```

**Comment by mariobyn** [comment]
Hi,
Let me try again and i will come back with an answer.

**Comment by mariobyn** [comment]
Try again now, it's working on my machine!

**Comment by mariobyn** [comment]
Get a pull before. You can use the jar file where you must create a WebSocket object and call run method in main method or where you want to use the library. 
Something like this:
WebSocket websocket = new Websocket();
websocket.run();

**Comment by SimonIsMe** [comment]
It's still not working. Browser's returning me the same error.
Here is my project: https://github.com/szymonides/MyWebServer.
I include the jar file into my project.

**Comment by mariobyn** [comment]
you can download all my project and create a main class where you will instantiate the WebSocket class like you did it. Then start the server and try again.

**Comment by SimonIsMe** [comment]
I did it. The problem is the same.
Also I tried connect with the server on Firefox 29 and it returned:

```
Firefox can't establish a connection to the server at ws://localhost/.
```

**Comment by mariobyn** [comment]
give me your email address to give you a working example.

**Comment by mariobyn** [comment]
Done. Check if it is working!

**Comment by SimonIsMe** [comment]
It's working. Thanks a lot for your help!

**Comment by mariobyn** [comment]
can you check what cause the problem?

**Comment by SimonIsMe** [comment]
I connected with the server in wrong way.
I used

```
var ws = new WebSocket('ws://localhost', 8112)
```

In your code is

```
var ws = new WebSocket('ws://localhost:8112')
```

and it's working

**Comment by mariobyn** [comment]
I'm glad that is working. Thanks.
